.env

```diff
@@ -1,3 +1,11 @@
-PORT=3000
+PORT=4000
+CORS_ORIGIN=https://localhost:3000
+
+TOKEN_KEY_ACCESS=WEATHER_USER_ACCESS
```

Comment:
Why did you add it as ENV variables?
The ENV variables are not secure.

It is better to use configuration json file.